src/Components/Blazor/Server/src/MonoDebugProxy/ws-proxy/MonoProxy.cs

```diff
@@ -461,7 +461,7 @@ async Task GetDetails(int msg_id, int object_id, CancellationToken token, string
 				{
 					result = var_list
 				});
-			} catch (Exception e) {
+			} catch (Exception) {
```

Comment:
no beuno. We should fix this in Mono and flow it through. But given how small these changes are, I think we can live with this for now.